modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Main.js

```diff
@@ -32,7 +32,7 @@ export default function Main({
 	trafficSources,
 }) {
 	return (
-		<>
+		<div className="c-p-3">
```

Comment:
This approach seems ok to me, but I imagine @eduardoallegrini and @marcoscv-work can speak to the relative merits of this approach vs the alternative, which would be to add a wrapper with a unique class of its own:

```js
<div className="sidebar-wrapper">...</div>
```

```scss
.sidebar-wrapper {
    padding: 1rem;
}
```

My understanding of the trade-off is:

- On the one hand, spacer utility classes (eg. `c-p-3`) sprinkled all over the codebase.
- Versus on the other, repeated custom classes that declare `padding`/`margin` etc over and over again.

I gather that we prefer the former (ie. your solution) because we can make global changes by changing the spacer definition in one place.

Reply:
What do you think @eduardoallegrini?

Reply:
I love it! 😄